dislib/data/array.py

```diff
@@ -645,7 +657,7 @@ def collect(self):
         return res
 
 
-def array(x, block_size):
+def array(x, block_size, **kwargs):
```

Comment:
Creating an Hecuba array from an existing table should be done in a `load_from_hecuba` function instead of here.

Reply:
Done
https://github.com/bsc-dd/dislib/blob/33795a0857a8b4ee5ecbe31228a8486cbc914112/dislib/data/array.py#L700-L733